src/components/donation/Steps/Result/Success/index.tsx

```diff
@@ -1,14 +1,14 @@
 import char from "assets/images/celebrating-character.png";
-import Image from "components/Image";
-import Signup from "components/Signup";
 import { appRoutes } from "constants/routes";
 import { confetti } from "helpers/confetti";
 import { Link } from "react-router-dom";
 import type { DonationRecipient } from "slices/donation";
 import { useGetter } from "store/accessors";
 import { userIsSignedIn } from "types/auth";
 import type { GuestDonor } from "types/aws";
-import Share, { type SocialMedia } from "./Share";
+import Image from "../../../../Image";
```

Comment:
nit: use relative import as this component uses another component in the same `src/components/*` folder